### PR TITLE
modifies the delly related commands and modified the related wdls and…

### PR DESCRIPTION
**Pull request by shadizaheri**
Here, I deleted the Delly related commands and modified the related wdls and the input template files (both for test and terra directories) accordingly.
**- FOR **dockerfiles** : I deleted the delly folder**

Next I tested the changes with womtool and it passed the validation.sh. After that to check for the possible conflicts that maybe the wamtool didn't catch, I tried running the pipeline with the following inputs:

- inputs/build/ref_panel_1kg/test/GatherSampleEvidence/GatherSampleEvidenceBatch.json
  cromshell job id: 5d473b14-6c3b-4434-81e7-5dee9ed42118
  status: **Succeeded**

- inputs/build/NA12878/test/GATKSVPipelineSingleSample.json
  cromshell job id: 061f9a70-9e5c-4451-a8ac-f83fdb383efd
  status: **Succeeded**

- inputs/build/ref_panel_1kg/test/GATKSVPipelineBatch/GATKSVPipelineBatch.FromSampleEvidence.json
  cromshell job id: a2844364-ce32-4274-b9cf-99556038bcf9
  status: **Succeeded**

### CODE REVIEW

**Comment by shadizaheri** [comment]
I created this PR, which I tested the changes with 3 inputs and it passed the jobs. This touches too many wdls and that’s why I thin it needs to be merged quickly in order to avoid merge conflicts. Any volunteers to review this please?
@VJalili @mwalker174 @cwhelan @epiercehoffman

**Review by VJalili — CHANGES_REQUESTED** [review]
This looks great, thank you @shadizaheri ! Delly references are mostly removed from the pipeline, there are only a few hits remaining (listed in the following) that I'm not sure if they should also be removed at this point; though the reference in `build_docker.py` should be removed since you are removing delly's Dockerfile. 

- `README`. In addition to removing delly, would it also be useful to add a comment on its deprecation?
- `inputs/templates/terra_workspaces/cohort_mode/cohort_mode_workspace_dashboard.md.tmpl`;
- `inputs/values/resources_hg38.json`;
- `scripts/cromwell/generate_inputs.py`;
- `scripts/docker/build_docker.py` (this should take care of the falling check `Docker Images / Test Images Build`;
- `scripts/docker/update_json_docker.sh`;
- `src/sv-pipeline/02_evidence_assessment/02a_rdtest/scripts/merge.sh`;
- `src/sv-pipeline/02_evidence_assessment/02a_rdtest/scripts/merge_allosomes.sh`; 
- `src/sv-pipeline/02_evidence_assessment/02a_rdtest/scripts/sub_merge_rdtest.sh`;
- `src/sv-pipeline/02_evidence_assessment/02a_rdtest/scripts/sub_rdtest_splits.sh`;
- `src/sv-pipeline/02_evidence_assessment/02a_rdtest/scripts/sub_rdtest_splits.XZ.sh`;
- `src/sv-pipeline/04_variant_resolution/scripts/overlap_pass.py`;
- `src/svtk/svtk/cli/standardize_vcf.py`;
- `src/svtk/svtk/standardize/__init__.py`;
- `src/svtk/svtk/std_delly.py`.

**Comment by cwhelan** [comment]
I'd leave the Delly standardizer in `svtk` just in case anyone wants to use that in the future. The scripts in 02 are really outdated and not used but I don't think you need to touch them in this PR; I'd just leave them for now.

**Review by cwhelan — CHANGES_REQUESTED** [review]
Looks good!

As Vahid pointed out, you should probably update the files in `inputs/` and `scripts/` that mention delly. 

Also, I think you've accidentally checked in your `dependencies.zip` which we don't want committed to master.

File: inputs/values/resources_hg38.json [hunk 1]
```diff
@@ -23,7 +22,6 @@
   "mei_bed" : "gs://gcp-public-data--broad-references/hg38/v0/sv-resources/resources/v1/mei_hg38.bed.gz",
   "melt_std_vcf_header" : "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/melt_standard_vcf_header.txt",
   "noncoding_bed" : "gs://gcp-public-data--broad-references/hg38/v0/sv-resources/resources/v1/noncoding.sort.hg38.bed",
-  "pesr_exclude_list" : "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/PESR.encode.peri_all.repeats.delly.hg38.blacklist.sorted.bed.gz",
```
Comment (epiercehoffman) [inline inputs/values/resources_hg38.json:26]: I know the file name has Delly in it, but this is used in several places unrelated to Delly, so we need to keep it. (And make sure to re-build the inputs and run the validate script)

Comment (shadizaheri) [reply inputs/values/resources_hg38.json]: @epiercehoffman , thanks for your review! I applied your comment in [this commit ](749aec92478cc198195e8ece013c630635f85525)

**Comment by shadizaheri** [comment]
Thank you @cwhelan and @VJalili for the great, prompt, and clear feedbacks!
I addressed your comments as listed below:
- `README`. This is right, it's better to justify why we are removing delly. The short answer would be because we don't use delly callers in this pipeline anymore. But about the detailed deprecation reasons, I think @epiercehoffman or @mwalker174 can comment better on that.
- `inputs/templates/terra_workspaces/cohort_mode/cohort_mode_workspace_dashboard.md.tmpl`; this was resolved by [this commit](https://github.com/broadinstitute/gatk-sv/pull/397/commits/eacb8fa2acd45188b98558bdcc4d30fe6f90891d)
- `inputs/values/resources_hg38.json`; this was resolved by [this commit](https://github.com/broadinstitute/gatk-sv/pull/397/commits/eacb8fa2acd45188b98558bdcc4d30fe6f90891d) and [this commit](https://github.com/broadinstitute/gatk-sv/pull/397/commits/749aec92478cc198195e8ece013c630635f85525)
- `scripts/cromwell/generate_inputs.py`; this was resolved by [this commit](https://github.com/broadinstitute/gatk-sv/pull/397/commits/eacb8fa2acd45188b98558bdcc4d30fe6f90891d)
- `scripts/docker/build_docker.py` (this should take care of the falling check `Docker Images / Test Images Build`;
- `scripts/docker/update_json_docker.sh`; this was resolved by [this commit](https://github.com/broadinstitute/gatk-sv/pull/397/commits/eacb8fa2acd45188b98558bdcc4d30fe6f90891d)
- `src/sv-pipeline/02_evidence_assessment/02a_rdtest/scripts/merge.sh`; following [Chris's comment](https://github.com/broadinstitute/gatk-sv/pull/397#issuecomment-1242074764) modifying 02 scripts possibly needs a separate PR.
- `src/sv-pipeline/02_evidence_assessment/02a_rdtest/scripts/merge_allosomes.sh`; following [Chris's comment](https://github.com/broadinstitute/gatk-sv/pull/397#issuecomment-1242074764) modifying 02 scripts possibly needs a separate PR.
- `src/sv-pipeline/02_evidence_assessment/02a_rdtest/scripts/sub_merge_rdtest.sh`; following [Chris's comment](https://github.com/broadinstitute/gatk-sv/pull/397#issuecomment-1242074764) modifying 02 scripts possibly needs a separate PR.
- `src/sv-pipeline/02_evidence_assessment/02a_rdtest/scripts/sub_rdtest_splits.sh`; following [Chris's comment](https://github.com/broadinstitute/gatk-sv/pull/397#issuecomment-1242074764) modifying 02 scripts possibly needs a separate PR.
- `src/sv-pipeline/02_evidence_assessment/02a_rdtest/scripts/sub_rdtest_splits.XZ.sh`; following [Chris's comment](https://github.com/broadinstitute/gatk-sv/pull/397#issuecomment-1242074764) modifying 02 scripts possibly needs a separate PR.
- `src/sv-pipeline/04_variant_resolution/scripts/overlap_pass.py`; delly is used [here](https://github.com/broadinstitute/gatk-sv/blob/master/src/sv-pipeline/04_variant_resolution/scripts/overlap_pass.py#:~:text=def-,_has_pesr,-(record)%3A) in defining a function. This function will be called in different places and simply removing it might cause problems. In my opinion, it is better if we modify  the `src/sv-pipeline/04_variant_resolution/scripts/overlap_pass.py` in a separate PR, maybe along with `src/sv-pipeline/02_evidence_assessment/02a_rdtest/scripts/sub_rdtest_splits.XZ.sh`. In this way we will be able to test the modifications to be sure that it didn't cause problems.
- `src/svtk/svtk/cli/standardize_vcf.py`; following [Chris's comment](https://github.com/broadinstitute/gatk-sv/pull/397#issuecomment-1242074764) SVTK and this commit are separated a bit.
- `src/svtk/svtk/standardize/__init__.py`; following [Chris's comment](https://github.com/broadinstitute/gatk-sv/pull/397#issuecomment-1242074764) SVTK and this commit are separated a bit.
- `src/svtk/svtk/std_delly.py`. following [Chris's comment](https://github.com/broadinstitute/gatk-sv/pull/397#issuecomment-1242074764) SVTK and this commit are separated a bit.
- I deleted the `dependecies.zip` in [this commit](https://github.com/broadinstitute/gatk-sv/pull/397/commits/f6998ed5d68d304e934bdd8f51c7eb3784a74b2a)

**Comment by shadizaheri** [comment]
In a conversation with @VJalili, he suggested that since @mwalker174 and @cwhelan were more involved in most of these wdls, maybe one/both of them can approve or reject this PR.

**Review by shadizaheri — COMMENTED** [review]
deleted comment

**Review by VJalili — APPROVED** [review]
LGTM, all my comments are addressed, thank you @shadizaheri!

**Review by cwhelan — APPROVED** [review]
This looks good, thanks for making the requested changes.

**Review by mwalker174 — APPROVED** [review]
Thanks @shadizaheri! This looks good to me as well. Congratulations on your first major PR!

There is one reference to Delly still in the README file on line 152 that can be removed. Otherwise, good to merge!

**Comment by shadizaheri** [comment]
@mwalker174 Thank you for reviewing the PR as well as all the help! I addressed the concern about the readme [in this commit](https://github.com/broadinstitute/gatk-sv/pull/397/commits/b5ab43ff1c0414a693ee8ef88ff23952bf2f42cd)

**Comment by shadizaheri** [comment]
Thank you all for reviewing this PR and for your helpful comments! 
I will merge the changes now.

**Pull request merged**